integration-tests/tests/sgx_payload.rs

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the SGX payload test.
+
+#![deny(clippy::all)]
+#![deny(missing_docs)]
+
+use std::path::Path;
+use std::process::Command;
+
+/// This test runs the payload in the SGX keep using the SGX shim.
+#[cfg_attr(not(has_sgx), ignore)]
+#[test]
+fn sgx_payload() {
+    // Define directories
+    let wksp_root = Path::new(env!("CARGO_MANIFEST_DIR"))
+        .parent()
+        .unwrap()
+        .to_str()
+        .unwrap();
```

Comment:
I wonder if we remove the `.to_str()` then just below turns into something like this:

```rust
let sgx_keep_dir = wksp_root.join("enarx-keep-sgx/target");
let payload_dir = wksp_root.join("payload/target/x86_64-unknown-linux-musl");
let sgx_shim_dir = wksp_root.join("enarx-keep-sgx-shim/target");
```

then I think we can do away with the `.to_owned()`s further down.